### ISSUE TITLE
Release Build Crash after Updating to 6.0.2

### ISSUE THREAD

**Issue by zjamshidi** [firebase/FirebaseUI-Android#1686]
I just updated to FirebaseUI 6.0.2. Debug build works fine but release build crashes.

> java.lang.RuntimeException: Facebook provider cannot be configured without dependency. Did you forget to add 'com.facebook.android:facebook-login:VERSION' dependency?

I have already included facebook login in .gradle file:
```
    implementation 'com.firebaseui:firebase-ui-auth:6.0.2'
    implementation 'com.facebook.android:facebook-login:5.5.1'
```
I downgraded to FirebaseUI version 5 again to resolve the crashes. Do you have any solution for it? Should I add anything to progaurd?

**Comment by samtstern** [comment]
Does this also happen if you use a 4.x version of the Facebook SDK?

**Comment by zjamshidi** [comment]
I tried the following dependecies:
```
    implementation 'com.firebaseui:firebase-ui-auth:6.0.2'
    implementation 'com.facebook.android:facebook-login:4.41.0'
```
and again got the crash:

> java.lang.RuntimeException: Facebook provider cannot be configured without dependency. Did you forget to add 'com.facebook.android:facebook-login:VERSION' dependency?

When I disabled the progaurd it works fine. Here is the progaurd's rules:
```
-keepattributes SourceFile,LineNumberTable

-renamesourcefileattribute SourceFile

-keepattributes *Annotation*

-keep class com.crashlytics.** { *; }
-dontwarn com.crashlytics.**

-keep class com.revenuecat.purchases.** { *; }
```
Should I add any thing to progaurd rules?

**Comment by willgurht** [comment]
In 6.02 i had to add 
# Keep the class names used to check for availablility
-keepnames class com.facebook.login.LoginManager

I guess AuthUI proguard file isnt applied properly since the update

**Comment by samtstern** [comment]
@willgurht thank you for sharing the fix!  That seems like the likely culprit, we had to change our proguarding after 6.0.0 so I bet the consumer rules got lose.

**Comment by zjamshidi** [comment]
Thanks @willgurht 
I will wait for the fix in 6.0.3

**Comment by hkchakladar** [comment]
Also facing this issue due to pro-guard.

**Comment by samtstern** [comment]
This has been released in FirebaseUI `6.1.0`
